lib/moonshot/stack_asg_printer.rb

```diff
@@ -111,13 +111,22 @@ def create_instance_table(asg_info)
     end
 
     def instance_row(asg_instance, ec2_instance)
+      public_ip = if ec2_instance
```

Comment:
Some thoughts:
- Rename 'public_ip', as it's not always public now.
- Leave a comment about why we think an instance might not be responding in EC2 (others have seen this too).
- I think 'offline' sounds too much like an understood condition, when really this appears during weird "eventually consistent" states at AWS. I'm thinking 'unknown' instead?

Maybe something like:

```ruby
if ec2_instance
  ip_address = ec2_instance.public_ip_address || "#{ec2_instance.private_ip_address} (PRV)"
  update = uptime_format(ec2_instance)
else
  # We've seen race conditions where ASG tells us about instances that EC2 is no longer
  # aware of.
  ip_address = 'unknown'
  uptime = 'unknown'
end